OrbitGl/App.cpp

```diff
@@ -270,8 +269,16 @@ void OrbitApp::PostInit() {
   }
 
   if (!options_.grpc_server_address.empty()) {
-    grpc_channel_ = grpc::CreateChannel(options_.grpc_server_address,
-                                        grpc::InsecureChannelCredentials());
+    grpc::ChannelArguments channel_arguments;
+    // TODO (antonrohr) move symbol loading to grpc stream.
+    // The default receive message size is 4mb. Symbol data can easily be more
+    // than this. This is set to an arbitrary size of 1500mb, which seems
+    // to be enough and leaves some headroom. As an example, a 1.1gb .debug
+    // symbols file results in a message size of 88mb.
+    channel_arguments.SetMaxReceiveMessageSize(1500 * 1024 * 1024);
```

Comment:
Going with -1 would also be ok I think. Or you can do std::numeric_limits<int32_t>::max() to get the maximum possible value that this method accepts.

Reply:
I rather go with the numeric limit.